### PR TITLE
Two more sig-cli tests and dashboards.

### PR DESCRIPTION
**Pull request by monopole**
One for kustomize presubmit tests on master, and the same for cli-utils.

### CODE REVIEW

File: config/testgrids/kubernetes/sig-cli/config.yaml [hunk 1]
```diff
@@ -19,3 +19,8 @@ dashboards:
       description: unit, integration and cmd tests for master branch
 
 - name: sig-cli-misc
+  dashboard_tab:
```
Comment (monopole) [inline config/testgrids/kubernetes/sig-cli/config.yaml]: This section might not be necessary given the annotations in the YAML in the jobs directory (not sure)

**Comment by chases2** [comment]
Are you migrating this information from somewhere else, or are you creating new tabs and groups? It looks like you're running into duplication issues.

If your Prow Jobs are [annotated](https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md#prow-job-configuration), do not also include the same Dashboard Tab and Test Group configuration. That will be read as a duplicate (since it's defined twice).

If you're adding a new dashboard, that does need to be added to the `config/testgrids` files. An annotation [creates a Test Group and Dashboard Tab](https://github.com/kubernetes/test-infra/tree/master/testgrid/cmd/configurator) for you.

File: config/testgrids/kubernetes/sig-cli/config.yaml (hunk 1 above)
Comment (chases2) [reply config/testgrids/kubernetes/sig-cli/config.yaml]: Yes, you'll need to delete this. This dashboard tab is defined twice (once here, and once as a Prow Annotation).

Keep the dashboard. `- name: sig-cli-misc`

**Comment by monopole** [comment]
Thanks Sean, i'll try to fix this later today.

**Comment by monopole** [comment]
OK, i think i've cargo-culted this sufficiently and it's good to go!

**Comment by monopole** [comment]
adding @krzyzacy for OWNERS approval, thanks Sen

File: config/jobs/kubernetes-sigs/cli-utils/cli-utils-presubmit-master.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    decorate: true
+    always_run: true
+    path_alias: sigs.k8s.io/cli-utils
+    branches:
+    - ^master$
+    spec:
+      containers:
+      - image: golang:1.13
+        command:
+        - make prow-presubmit-check
```
Comment (krzyzacy) [inline config/jobs/kubernetes-sigs/cli-utils/cli-utils-presubmit-master.yaml]: I think they need to be in separate lines :-)

File: config/jobs/kubernetes-sigs/kustomize/kustomize-presubmit-master.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    decorate: true
+    always_run: true
+    path_alias: sigs.k8s.io/kustomize
+    branches:
+    - ^master$
+    spec:
+      containers:
+      - image: golang:1.13
+        command:
+        - make prow-presubmit-check
```
Comment (krzyzacy) [inline config/jobs/kubernetes-sigs/kustomize/kustomize-presubmit-master.yaml]: ditto

File: config/jobs/kubernetes-sigs/cli-utils/cli-utils-presubmit-master.yaml (hunk 2 above)
Comment (monopole) [reply config/jobs/kubernetes-sigs/cli-utils/cli-utils-presubmit-master.yaml]: done, thanks

File: config/jobs/kubernetes-sigs/kustomize/kustomize-presubmit-master.yaml (hunk 3 above)
Comment (monopole) [reply config/jobs/kubernetes-sigs/kustomize/kustomize-presubmit-master.yaml]: done thanks!

**Pull request merged**